src/config.rs

```diff
@@ -0,0 +1,32 @@
+use config::{Config, ConfigError};
```

Comment:
Can we move it into its folder? I mean, have `config/mod.rs` instead of `config.rs`.